requirements-pants.txt

```diff
@@ -8,6 +8,7 @@
 
 apscheduler
 argcomplete
+argparse
```

Comment:
This was only included as a transitive dep of the old version of oslo.config, and the newer versions don't require it, so pants happily removed it from the lockfile. However, our code directly imports it, so I added it back as a direct dep.